features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/apis/v1/asset_api_router.jag

```diff
@@ -511,190 +520,200 @@ require('/modules/publisher.js').exec(function(ctx) {
+                artifact.attributes["overview_appTenant"] = appTenant;
+                artifact.attributes["overview_advertiseOnly"] = isAdvertiseOnly;
+
+
+                var tenantId = server.current(session).tenantId;
+                var tenantDomain = carbon.server.tenantDomain({tenantId: tenantId});
+                var subscriptionAvailability = (artifact.attributes["overview_subscriptionAvailability"]).trim();
+                if (subscriptionAvailability == "current_tenant") {
+                    artifact.attributes["overview_tenants"] = tenantDomain;
+                }
 
-            if (subscriptionAvailability == "all_tenants") {
-                artifact.attributes["overview_tenants"] = "";
-            }
-            var saml2SsoIssuer = null;
-            if (tenantId == -1234) {
-                saml2SsoIssuer = artifact.attributes["overview_name"] + "-" + artifact.attributes["overview_version"];
-            } else {
-                saml2SsoIssuer = artifact.attributes["overview_name"] + "-" + tenantDomain + "-" + artifact.attributes["overview_version"];
-            }
-            // update sso detail
-            var idpProviderUrl = dataConfigs.ssoConfiguration.identityProviderURL;
-            var ssoEnabled = dataConfigs.ssoConfiguration.enabled;
-            artifact.attributes["sso_idpProviderUrl"] = idpProviderUrl;
-            artifact.attributes["sso_saml2SsoIssuer"] = saml2SsoIssuer;
-            if (ssoEnabled) {
-                artifact.attributes["sso_singleSignOn"] = "Enabled";
-            } else {
-                artifact.attributes["sso_singleSignOn"] = "Disabled";
-            }
+                if (subscriptionAvailability == "all_tenants") {
+                    artifact.attributes["overview_tenants"] = "";
+                }
+                var saml2SsoIssuer = null;
+                if (tenantId == -1234) {
+                    saml2SsoIssuer =
+                    artifact.attributes["overview_name"] + "-" + artifact.attributes["overview_version"];
+                } else {
+                    saml2SsoIssuer = artifact.attributes["overview_name"] + "-" + tenantDomain + "-"
+                                     + artifact.attributes["overview_version"];
+                }
+                // update sso detail
+                var idpProviderUrl = dataConfigs.ssoConfiguration.identityProviderURL;
+                var ssoEnabled = dataConfigs.ssoConfiguration.enabled;
+                artifact.attributes["sso_idpProviderUrl"] = idpProviderUrl;
+                artifact.attributes["sso_saml2SsoIssuer"] = saml2SsoIssuer;
+                if (ssoEnabled) {
+                    artifact.attributes["sso_singleSignOn"] = "Enabled";
+                } else {
+                    artifact.attributes["sso_singleSignOn"] = "Disabled";
+                }
 
 
-            var index = 0;
-            var uritemplate = "uritemplate_urlPattern" + index;
+                var index = 0;
```

Comment:
There are several formatting issues. Please fix all places. Also note that you should not modify or format other codes addition to your changes.